src/client/app/auth/api.service.js

```diff
@@ -8,23 +8,18 @@
     AuthApi.$inject = ['$http'];
 
     function AuthApi($http) {
-        var service = {
-            login: buildPostRequest('/api/auth/login')
-            //logoff: buildPostRequest('/api/auth/logoff') tbd
+        return {
+            authentificateUser: function(credentials) {
+                $http.post('/api/auth', credentials)
+                    .success(function (data, status, headers) {
+                        var token = headers(["x-auth-token"]);
+                        $http.get('/api/users/current', {
+                            headers: {
```

Comment:
this is not needed, you should use interceptor to add a token as a default header to all requests one the user is authenticated